jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -271,6 +271,9 @@ YUI.add('user-profile', function() {
         return;
       }
       var listItems = [];
+      if (typeof series !== 'array') {
```

Comment:
This will always be evaluated because `typeof [] === 'object'`

You want `if (Array.isArray(series)) {`